### ISSUE TITLE
Error running typst-test after init: unexpected character '\n' after patch version number

### ISSUE THREAD

**Issue by keyneston** [tingerrr/typst-test#19]
Error:

```
$ typst-test update
typst-test ran into an unexpected error, this is most likely a bug
Please consider reporting this at https://github.com/tingerrr/typst-test/issues/new
```

`typst.toml`

```toml
[package]
name = "mypackage"
version = "0.0.2"
entrypoint = "lib.typ"
authors = ["Keyneston"]
repository = "https://github.com/keyneston/mypackage"
license = "MIT"
```

**Comment by keyneston** [comment]
This is with HEAD. Switching cargo to install `ci-semi-stable` resolved the issue.

**Comment by tingerrr** [comment]
I tried it locally (5d2574e7) with the manifest you sent, it's missing the required field `description`. Unfortunately, it does not tell the user why the manifest is wrong.

But I can't reproduce the error you sent, it simply warns that the manifest is invalid. I'll push a fix real quick.

> This is with HEAD. Switching cargo to install `ci-semi-stable` resolved the issue.

The tag, I assume, I recently switched to a branch of the same name as this is easier to continuously bump.
